examples/chip-tool/commands/clusters/ModelCommand.cpp

```diff
@@ -54,6 +73,13 @@ CHIP_ERROR ModelCommand::Run(NodeId localId, NodeId remoteId)
         err = GetExecContext()->commissioner->GetDevice(remoteId, &mDevice);
         VerifyOrExit(err == CHIP_NO_ERROR, ChipLogError(chipTool, "Init failure! No pairing for device: %" PRIu64, localId));
 
+        if (mDevice->IsSessionSetupInProgress())
```

Comment:
if  does not seem needed: WaitForSessionSetup already has the condition internally and the for loop will exit immediately if it is met.